contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -397,77 +511,50 @@ Already have a warehouse? You can [link it with ours](/docs/data-warehouse) to a
-| 5k                        | $124                 | $0          | <strong class="text-green">$1,488 per year cheaper</strong>  |
-| 10k                       | $186                 | $0          | <strong class="text-green">$2,232 per year cheaper</strong>  |
-| 25k                       | $249                 | $217        | <strong class="text-green">$1,128 per year cheaper</strong>  |
-| 50k                       | $561                 | $430        | <strong class="text-green">$1,572 per year cheaper</strong>  |
-| 75k                       | $841                 | $625        | <strong class="text-green">$2,592 per year cheaper</strong>  |
-| 100k                      | $1,061               | $820        | <strong class="text-green">$2,892 per year cheaper</strong>  |
-| 150k                      | $1,584               | $1,207      | <strong class="text-green">$4,524 per year cheaper</strong>  |
-| 200k                      | $2,100               | $1,600      | <strong class="text-green">$6,000 per year cheaper</strong>  |
-| 300k                      | $3,150               | $2,091      | <strong class="text-green">$12,708 per year cheaper</strong> |
+**PostHog charges based on usage**. Each product (analytics, session replay, feature flags, etc.) comes with a [generous free tier](/pricing), and once you exceed those limits you pay only for what you use. Pricing is fully transparent and published publicly, with per-unit rates you can calculate in advance. You can also set billing caps per product to avoid unexpected costs.
 
-We also provide numerous ways to [control how many events you send](/tutorials/fewer-unwanted-events) and reduce your costs, including the option to send cheaper **anonymous events**.
+This usage-based approach means costs scale with your actual activity, whether that's events, surveys, or recordings, rather than the number of users in your product.
 
-### Identified vs anonymous events
+### Amplitude pricing philosophy
 
-Anonymous events **only include non-identifying data**, such as referral data (UTMs, domains, etc), device type, and basic location information.
+**Amplitude charges based on Monthly Tracked Users (MTUs)**. Each tracked user is counted once per month, regardless of how many events they generate. Plans scale as your MTUs grow, with higher tiers unlocking more advanced analytics, experimentation, and enterprise features.
 
-They're useful if you're tracking a huge number of users and only want to analyze aggregate data, such as on a content website with millions of visitors.
+This model can be easier to predict if you run a large product with millions of users who generate relatively few events per person. It also means teams don't have to think about event volume when forecasting costs. However, it can feel less flexible if you're working with a smaller but more active user base, since a few highly engaged users won't change your MTU count.
```

Comment:
```suggestion
This model can be easier to predict if you run a large product with millions of users, but it's less flexible because there's no way to reduce how much you spend. Every user is counted equally regardless of how valuable they are to you.
```

@ivanagas interested in your take here – I think this is a better way describe this difference.